project-scratch-def/project-scratch-def.schema.json

```diff
@@ -1,6 +1,5 @@
 {
```

Comment:
We can't do the folder rename yet (removing `.json`) since the vscode extension refers to the master version. Undo that change and we can talk about how to migrate it over in another PR.

Reply:
Ah. Ok. I was wondering if there was more to that than just an oversight.